models/issues/pull_list.go

```diff
@@ -46,6 +47,10 @@ func listPullRequestStatement(ctx context.Context, baseRepoID int64, opts *PullR
 		sess.And("issue.milestone_id=?", opts.MilestoneID)
 	}
 
+	if opts.OwnerID > 0 {
+		sess.And("issue.owner_id=?", opts.OwnerID)
```

Comment:
I don't think issue has the column `owner_id`.

Reply:
good catch misslooked while looking at the other api ...